Tests/SmithyIdentityTests/CustomAWSCredentialIdentityResolverTests.swift

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
This test was moved from aws-sdk-swift to here, as the thing it's testing was moved as well.